content/concept-amendments.md

```diff
@@ -86,13 +86,122 @@ When an amendment gets enabled for the network after the voting process, servers
 * Do not participate in the consensus process
 * Do not vote on future amendments
 
-Becoming amendment blocked is a security feature to protect backend applications. Rather than guessing and maybe misinterpreting a ledger after new rules have applied, `rippled` reports that it does not know the state of the ledger because it does not know how the amendment works.
+Becoming amendment blocked is a security feature to protect backend applications. Rather than guessing and maybe misinterpreting a ledger after new rules have been applied, `rippled` reports that it does not know the state of the ledger because it does not know how the amendment works.
```

Comment:
Is this change necessary? Rules can "apply", active voice, can't they?

Reply:
I think the question might be what's applying the rules to what? My interpretation is that In the original, the rules are the agent applying themselves to the ledger. In the amended version, `rippled` has applied the rules to the ledger. FWIW, the amended version makes more intuitive sense to me as a (maybe not very smart) reader.